datadog_operator/assets/dashboards/operator_overview.json

```
@@ -0,0 +1,1987 @@
+                                            "name": "query2"
+                                        },
+                                        {
+                                            "query": "sum:datadog.operator.controller_runtime_reconcile_total{$scope,$cluster,kube_app_name:datadog-operator,result:requeue} by {kube_cluster_name}.as_count()",
+                                            "data_source": "metrics",
+                                            "name": "query3"
+                                        },
+                                        {
+                                            "query": "sum:datadog.operator.controller_runtime_reconcile_total{$scope,$cluster,kube_app_name:datadog-operator,result:requeue_after} by {kube_cluster_name}.as_count()",
+                                            "data_source": "metrics",
+                                            "name": "query4"
+                                        }
+                                    ],
+                                    "style": {
+                                        "palette": "dog_classic",
+                                        "line_type": "solid",
+                                        "line_width": "normal"
+                                    },
+                                    "display_type": "line"
+                                }
+                            ]
+                        },
+                        "layout": {
+                            "x": 4,
+                            "y": 0,
+                            "width": 4,
+                            "height": 3
+                        }
+                    },
+                    {
+                        "id": 5927604654245473,
+                        "definition": {
+                            "title": "Reconcile errors total",
+                            "show_legend": true,
+                            "legend_layout": "auto",
+                            "legend_columns": [
+                                "avg",
+                                "min",
+                                "max",
+                                "value",
+                                "sum"
+                            ],
+                            "type": "timeseries",
+                            "requests": [
+                                {
+                                    "formulas": [
+                                        {
+                                            "formula": "query1"
+                                        }
+                                    ],
+                                    "response_format": "timeseries",
+                                    "queries": [
+                                        {
+                                            "query": "avg:datadog.operator.controller_runtime_reconcile_errors_total{$scope,$cluster,kube_app_name:datadog-operator} by {kube_cluster_name}.as_count()",
```

```suggestion
                                            "query": "count:datadog.operator.controller_runtime_reconcile_errors_total{$scope,$cluster,kube_app_name:datadog-operator} by {kube_cluster_name}.as_count()",
```

I think is a better aggregation